membership-attribute-service/app/services/AttributesFromZuora.scala

```diff
@@ -31,26 +34,30 @@ class AttributesFromZuora(implicit val executionContext: ExecutionContext) exten
       attributes
     }
 
-    val zuoraAttributesDisjunction: DisjunctionT[Future, String, Option[ZuoraAttributes]] = for {
+    def getResultIf[R](shouldCallFunction: Boolean, whichCall: String, futureResult: Future[Disjunction[String, R]], default: R, identityId: String): Future[Disjunction[String, R]] = {
```

Comment:
do you need identity id and default as params here?
it seems like it always uses the identity id in scope and nil as the default